lightning/src/ln/msgs.rs

```diff
@@ -441,6 +441,53 @@ pub struct ChannelReady {
 	pub short_channel_id_alias: Option<u64>,
 }
 
+/// A splice message to be sent by or received from the splice initiator.
+/// TODO(splicing): Is using 'splice initiator' role OK?
```

Comment:
This must correspond to the `stfu` [`initiator`](https://github.com/lightning/bolts/blob/a68d7b07cea0fb243f25e955cca6134cae5872cf/02-peer-protocol.md#stfu) field. That's why I think we should include the `stfu` message here too.

Or is there a different question here? :)

Reply:
OK